lesson_26/script.js

```diff
@@ -0,0 +1,67 @@
+
+function Accumulator(initialValue) {
+    this.value = initialValue;
+    this.increment = function() {
+        this.value++;
+    };
+    this.decrement = function() {
+        this.value--;
+    };
+}
+
+function CancelableAccumulator(initialValue) {
+    Accumulator.call(this, initialValue);
+    this.originalValue = initialValue;
+    this.clear = function() {
+        this.value = this.originalValue;
+    };
+}
```

Comment:
and you never run this functions and don't create objects from them to see that it works:(